### ISSUE TITLE
Upload not working on one file

### ISSUE THREAD

**Issue by johnall** [uglyrobot/big-file-uploads#1]
I have an Upload not working on one file. It's a video that is zipped and its size is 23.9 MB. All others are working, this onegets to about 90% and stops with an error. Any Ideas

**Comment by andtrev** [comment]
zip files aren't allowed by the WordPress media uploader by default, you can see this link for more info on how to add some code to enable new file types http://www.wpbeginner.com/wp-tutorials/how-to-add-additional-file-types-to-be-uploaded-in-wordpress/ , or I believe this plugin will still work https://wordpress.org/plugins/upload-file-type-settings-plugin/

**Comment by johnall** [comment]
Hi,
I have WordPress 4.4.2 and that is allowing me to upload zip files.Your plug in is working fine. I have been uploading other zip files but I was only having trouble with one.I did upload it via FTP but could not get it to link to my product. I finally figured how to link it and that file is working fine now as well as all the others that i uploaded using your plug in.It must be something freaky with that one file.Your answer to my question gave me a clue as to how to solve my problem and I thank you for that!
